crates/oxc_cli/src/lib.rs

```diff
@@ -29,9 +45,17 @@ impl Cli {
             .par_iter()
             .map(|path| {
                 let diagnostics = Self::lint_path(path);
-                for diagnostic in &diagnostics {
-                    println!("{diagnostic:?}");
-                }
+
+                diagnostics
+                    .iter()
+                    .filter(|diagnostic| {
+                        diagnostic.severity().unwrap() != Severity::Warning
```

Comment:
Perhaps the comment 

> // This flag follows ESLint's --quiet behavior as documented here: https://eslint.org/docs/latest/use/command-line-interface#--quiet
> // Note that it does not disable ALL diagnostics, only Warning diagnostics

should be put here, this is confusing as hell lol

Reply:
Ah yeah I added it in `command.rs`, ill move it to here